draft-ietf-rats-architecture.md

```diff
@@ -842,40 +842,47 @@ a different format.
-The first approach is that a nonce is generated by a remote entity
-(e.g., the Verifier for Evidence, or the Relying Party for an Attestation
-Result), and the nonce is then signed and included along with the claims in
-the Evidence or Attestation Result, so that the remote entity knows that the
-claims were signed after the nonce was generated.
-
-A second approach is to rely on synchronized clocks, and include a signed
-timestamp (e.g., using {{-rats-tuda}}) along with the
-claims in the Evidence or Attestation Result, so that the remote entity knows
-that the claims were signed at that time, as long as it has some assurance that
-the timestamp is correct.  This typically requires additional claims about
-the signer's time synchronization mechanism in order to provide such assurance.
-
-In either approach, it is important to note that the actual values in claims
-might have been generated long before the claims are signed.  If so, it is the
-signer's responsibility to ensure that the values are still correct when they
-are signed.  For example, values might have been generated at boot, and then
-used in claims as long as the signer can guarantee that they cannot have changed
-since boot.
+# Freshness
+
+A remote entity (Verifier or Relying Party) may need to learn the point in
+time (i.e., the "epoch") an Evidence or Attestation Result have been
+produced.
+This is essential in deciding whether the included Claims and their values can be considered fresh, meaning they still reflect the latest state of the Attester, and that any Attestation Result was generated using the latest Appraisal Policy for Evidence.
+
+Freshness is assessed based on a policy defined by the consuming entity,
+Verifier or Relying Party, that compares the estimated epoch against an
+"expiry" threshold defined locally to that policy.
+There is, however, always a race condition possible in that the state of the Attester, and the Appraisal Policy for Evidence, might change immediately after the Evidence or Attestation Result was generated.
+The goal is merely to narrow their recentness to something the Verifier (for Evidence) or Relying Party (for Attestation Result) is willing to accept.
+Freshness is a key component  for enabling caching and reuse of both Evidence and Attestation Results, which is especially valuable in cases where their computation uses a substantial part of the resource budget (e.g., energy in constrained devices).
+
+There are two common approaches for determining the epoch of an Evidence or Attestation Result.
+
+<!-- Explicit Timekeeping using Synchronized Clocks -->
+
+The first approach is to rely on synchronized and trustworthy clocks, and include a signed timestamp (see {{?I-D.birkholz-rats-tuda}}) along with the Claims in the Evidence or Attestation Result.
+Timestamps can also be added on a per-Claim basis if wanted, to allow distinguishing the time of creation of Evidence from the time that a specific Claim value was generated.
```

Comment:
"also", "if wanted", "of Evidence", "value" are not needed.
"allow distinguishing" change to "distinguish"

Reply:
fixed in c215569